src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        File file = new File(filePath);
+        try (BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(file))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                lines.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("can not read file: " + filePath, e);
```

Comment:
When rethrowing the exception as a RuntimeException, make sure to use an informative message. In this case, the message should start with a capital letter.